tests/e2e/src/steps/portal.steps.ts

```diff
@@ -169,6 +169,12 @@ Given(
   }
 );
 
+Given("I fill the address input field with {string}", async function (this: ICustomWorld, inputText: string) {
```

Comment:
I suppose we can reuse `I fill the text field for {string} selector by {string} value` from the portal.steps.ts